### PR TITLE
REL-4303: Flamingos-2 R3000 grating throughput

### PR DESCRIPTION
**Pull request by andrewwstephens**
This updates the Flamingos-2 R3000 grating efficiency curves to better match observations.
The old curves are shown by the dashed lines (the green m=5 curve was missing).
The measurements are the dots.
The new curves are the solid lines.
R3K

### CODE REVIEW

**Comment by cquiroz** [comment]
Will the dashed lines be shown on the itc graphs?

File: bundle/edu.gemini.itc.web/src/test/scala/edu/gemini/itc/web/baseline/BaselineTest.scala [hunk 1]
```diff
@@ -41,7 +41,7 @@ class BaselineTest {
     Baseline.write(baseSeq)
   }
 
-  //@Ignore
+  @Ignore
```
Comment (cquiroz) [inline bundle/edu.gemini.itc.web/src/test/scala/edu/gemini/itc/web/baseline/BaselineTest.scala:44]: This is a fairly slow test, maybe me keep it commented?

**Review by cquiroz — APPROVED** [review]
LGTM

File: bundle/edu.gemini.itc.web/src/test/scala/edu/gemini/itc/web/baseline/BaselineTest.scala (hunk 1 above)
Comment (cquiroz) [reply bundle/edu.gemini.itc.web/src/test/scala/edu/gemini/itc/web/baseline/BaselineTest.scala:44]: Sorry I got confused, we actually want to Ignore this test

**Comment by andrewwstephens** [comment]
The dashed lines are only to inform those who will be testing the changes - they will not be visible in the iTC.

**Pull request merged**